src/rqt_graph/ros_graph.py

```diff
@@ -398,7 +398,7 @@ def _load_dot(self, file_name=None):
         self._widget.group_image_check_box.setEnabled(False)
         self._widget.hide_dynamic_reconfigure_check_box.setEnabled(False)
 
-        self._update_graph_view(dotcode)
+        self._update_graph_view(dotcode.decode('utf-8'))
```

Comment:
This change has already been applied to the `rolling` branhc.

Reply:
Might be worth deleting the ros2 branch.